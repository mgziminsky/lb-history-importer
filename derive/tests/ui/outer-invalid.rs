use lb_importer_derive::IntoPayload;

#[derive(IntoPayload)]
#[payload(track = test: String)]
#[payload(invalid = other: String)]
struct MultipleOuter {
    test: String,
    other: String,
}

fn main() {}
