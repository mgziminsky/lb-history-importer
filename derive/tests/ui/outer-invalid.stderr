error: Invalid field name; expected `track`, `artist`, or `release`
 --> tests/ui/outer-invalid.rs:5:11
  |
5 | #[payload(invalid = other: String)]
  |           ^^^^^^^
